src/niobot/client.py

```diff
@@ -769,9 +769,27 @@ async def create_dm_room(
         result = await self.room_create(
             is_direct=True,
             invite=[user_id],
+            visibility=nio.api.RoomVisibility.private,
+            preset=nio.api.RoomPreset.trusted_private_chat,
         )
         if isinstance(result, nio.RoomCreateError):
             raise GenericMatrixError("Failed to create DM room", response=result)
+        
+        dm_rooms = {}
+        dm_rooms[user_id] = [result.room_id]
+
+        logging.debug(f"create_dm_user: user_id {user_id} self.user_id {self.user_id}")
+        # Trying to send m.direct type eventfrom nio.responses import DirectRoomsErrorResponse, DirectRoomsResponse
+        logging.debug(f"create_dm_user: setting m.direct type with rooms {nio.Api.to_json(dm_rooms)}")
+        await self._send(
+            nio.DirectRoomsResponse,
+            'PUT',
+            nio.Api._build_path(    ["user", self.user_id, "account_data", "m.direct"],
```

Comment:
The whitespace is a little odd here - could you run `ruff check --fix` and `ruff format` & push on your fork? just for styling consistency